python/pyspark/sql/streaming/readwriter.py

```diff
@@ -641,8 +641,8 @@ def csv(
 
         Parameters
         ----------
-        path : str or list
```

Comment:
Just curious, what's the error message if you pass a list here?

Reply:
It will throw a PySparkTypeError saying argument passed is not string

Reply:
Yes, it explicitly checks if the argument is a string:
https://github.com/apache/spark/blob/master/python/pyspark/sql/streaming/readwriter.py#L717-L723

Reply:
Got it thanks!